### PR TITLE
add timings to the build script

### PR DESCRIPTION
**Pull request by asmega**
- my limited bash knowledge to add timings to steps of the build process to identify bottlenecks

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -18,38 +18,86 @@ cat > public/version <<EOT
...
+echo "================="
+
+START_TIME=$SECONDS
 echo "Purging bower components"
 echo "----"
 [ -d vendor/assets/bower_components ] && rm -r vendor/assets/bower_components
 
+ELAPSED_TIME=$(($SECONDS - $START_TIME))
+echo "================="
+echo "Purging bower componments took $ELAPSED_TIME seconds"
```
Comment (Tr4pSt3R) [inline build.sh]: typo *components

File: build.sh [hunk 2]
```diff
@@ -18,38 +18,86 @@ cat > public/version <<EOT
...
 cd ..
 /usr/local/rpm_builder/create-rails-rpm $artifact_name $artifact_name $version
 
+ELAPSED_TIME=$(($SECONDS - $START_TIME))
+echo "================="
+echo "RPM took $ELAPSED_TIME seconds"
+echo "================="
+
 #prune packaged gems
 rm -rf vendor/cache
```
Comment (tomas-stefano) [inline build.sh]: `time /usr/local/rpm_builder/create-rails-rpm $artifact_name $artifact_name $version` will work right?

File: build.sh (hunk 1 above)
Comment (asmega) [inline build.sh]: fixed

**Comment by Tr4pSt3R** [comment]
:+1: :100:

File: build.sh (hunk 2 above)
Comment (asmega) [inline build.sh]: yeah that's a better idea

Comment (asmega) [inline build.sh]: how about now? i only timed the longer running commands

**Comment by tomas-stefano** [comment]
:+1: :ship:

**Pull request merged**